exam/backend/server.js

```diff
@@ -0,0 +1,28 @@
+const express = require('express') ;
+const cors = require('cors') ;
+require('dotenv').config() ;
+const app = express() ;
+const port = process.env.PORT || 5000 ;
+
+app.use(cors()) ;
+app.use(express.json()) ;
+const db = require('./models') ;
+db.sequelize.sync({force:false}).then( () =>{
+  console.log('Database works fine') ;
+}).catch( (err) =>{
+  console.log('Error' + err ) ;
+}) ;
+
+// testing addition of fields
+// const jane = db.user.create({name: 'Jane', email: 'Doelana' , password: 'HASH' , role:1   }).then( (user) =>{
+//   console.log('done' ) ;
+// }).catch( (err) =>{
+//   console.error('There is an error ' + err ) ;
+// });
+
+const register_router = require('./routes/register') ;
+// const login_router = require('./routes/login')
+app.use('/register' , register_router ) ;
```

Comment:
It seems to be an overKill, we can have a  router as User and add register as a route there?

Reply:
Legit point , will fix it .